babycenter/static/css/modules/_article-list.scss

```diff
@@ -13,7 +14,8 @@
 
 .article-list__title {
   font-weight: bold;
-  display: inline;
+  display: block;
+
 
   &:after {
```

Comment:
Do you think it still makes sense to add the colon `:` if the titles are not on the same line?